### PR TITLE
Add translations to the project builder

### LINKED ISSUE #4323: Project builder: add translation listing and controls
Project owners should see translations for their projects listed in the project builder. They should have links, to review the translation, and checkboxes to publish/remove the translation from the project. Ideally, they should be able to preview a translation in PFE before publishing it.

### PR DESCRIPTION
**Pull request by eatyourgreens**
Adds a basic translations page to the project builder, with links to edit and preview your project's translations.

Staging branch URL: https://translations.pfe-preview.zooniverse.org

Fixes #4323.

Todo:
 - [x] Add a translations page
 - [x] Connect up the project builder to the redux store
 - [x] List available languages with links to preview your project in that language, and also to edit that language.

Not covered in this PR: adding a mechanism to make translations public for a project, once they are complete.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Comment by eatyourgreens** [comment]
This should be ready to review now.

Galaxy Zoo is an example of a project that has translations:
https://translations.pfe-preview.zooniverse.org/lab/5733?env=production

File: app/pages/lab/translations/index.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+TranslationsManager.defaultProps = {
+  translations: {
+    languages: {
+      project: []
+    }
+  }
+};
+
+export default connect(mapStateToProps, mapDispatchToProps)(TranslationsManager);
+export { TranslationsManager };
```
Comment (wgranger) [inline app/pages/lab/translations/index.jsx]: Should there be two exports here?

Comment (eatyourgreens) [reply app/pages/lab/translations/index.jsx]: The bare component is exported in case its needed for tests etc.

**Comment by eatyourgreens** [comment]
@wgranger looks like Google Analytics has added `&ga=…` to the edit link. I don't see that, but I've got an ad blocker turned on.

**Comment by eatyourgreens** [comment]
Yeah, query params appear in a URL before the fragment identifier:
`https://zooniverse.org/?env=production&language=en#projects` points to the same location as `https://zooniverse.org/#projects`.

React router, for some reason, doesn't understand this and breaks URLs by adding query params after the hash character.

**Comment by eatyourgreens** [comment]
External links seem to be fixed by adding a ? to the URL, to fake an empty query string. I've rebased and restaged this.

The aim of this PR is to have a page up so that Scribes and Galaxy Zoo can quickly see which languages have been added to their projects. I'm happy to leave creating translations (when you don't have any yet) until a future PR.

**Comment by wgranger** [comment]
Great. This looks good for a merge on Monday.

**Pull request merged**